src/include/sof/audio/component.h

```diff
@@ -236,12 +236,14 @@ enum {
 		  (uint32_t)((pcd)->cpu_delta_peak))
 
 #define comp_perf_avg_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u %u",\
+	comp_info(comp_p, "perf comp_copy samples %u period %u cpu avg %u peak %u %u %u %u",\
```

Comment:
can we at least add iobs in the log?

`perf comp_copy samples %u period %u cpu avg %u peak %u %u iobs %u %u`

Reply:
just want to minimize the change for sof and script both, since parsing can directly pick ibs obs with a fixed offset.

Reply:
But readability is also important, no one knows what they are until looking and understanding the code. Even you add iobs, the values are still at some fixed offset.

Reply:
is the "old" (xtos / sof-logger) limit of a maximum of 4 variables in logs now completely abandoned? What about all the platforms still using XTOS?

Reply:
@lgirdwood , yes it is runtime read out from each module.

@lyakh , this is good point, I did not test IPC3 case, if consider IPC3 case, we may need separate logs to two lines or remove some variable print.